astropy/coordinates/baseframe.py

```diff
@@ -30,11 +30,15 @@
                              SphericalRepresentation,
                              UnitSphericalRepresentation,
                              REPRESENTATION_CLASSES)
+from .frame_attributes import FrameAttribute
 
-# Import all FrameAttributes so we don't break backwards-compatibility
+# Import all orher FrameAttributes so we don't break backwards-compatibility
 # (some users rely on them being here, although that is not
 # encouraged, as this is not the public API location.)
-from .frame_attributes import *
+from .frame_attributes import (
```

Comment:
Why did you change this back? The argument for * was that we then don't have to remember to update this if we add any more `FrameAttribute` classes...